src/Microsoft.ML.FastTree/GamTrainer.cs

```diff
@@ -34,8 +34,33 @@ namespace Microsoft.ML.Trainers.FastTree
     using SplitInfo = LeastSquaresRegressionTreeLearner.SplitInfo;
 
     /// <summary>
-    /// Generalized Additive Model Learner.
+    /// Generalized Additive Model Trainer.
     /// </summary>
+    /// <remarks>
+    /// <para>
+    /// Generalized Additive Models, or GAMs, model the data as an
+    /// independent set of feature responses similar to a linear model. For each feature, the GAM trainer
+    /// learns a function, known as a shape function, that outputs a response as a function
+    /// of the feature value. (In contrast, a linear model fits a linear response (e.g. a line) to each feature).
```

Comment:
There are several ambiguous terms, for example, `shape function`, `response`, and `independent set`. Are all those terms consistent with the paper referenced here? Not sure if users would be happy if there is a simple regression example to explain its prediction rule (not training algorithm).

Reply:
Thanks for the close read! `shape function` and `response` are consistent with the literature. `independent set` was not -- I changed this to `linearly independent`. That was what I was looking for :)